Polygon.js

```diff
@@ -0,0 +1,85 @@
+{
+	"translatorID": "fa7c37b1-fda4-418a-a8b8-2491929411ab",
+	"label": "Polygon",
+	"creator": "czar",
+	"target": "^https?://(?:www\\.)polygon\\.com",
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2015-12-05 20:54:17"
+}
+
+// http://en.wikipedia.org/wiki/User_talk:Czar
+
+function detectWeb(doc, url) {
+	if (doc.title.match("| Polygon")) {
+ 		return "webpage";
+ 	}
+	
+	return null;
+}
+
+function scrape(doc, url) {
+	var namespace = doc.documentElement.namespaceURI; 
+	var nsResolver = namespace ? function(prefix) {
+		if (prefix == 'x') return namespace; else return null; 
+		} : null; 
+	
+	var newItem = new Zotero.Item("webpage");
+	newItem.url = doc.location.href;
+	newItem.language = "en";
+	newItem.publicationTitle="Polygon";
+	
+		// Article title
+	var soughtXPath = "//h2";
+		evalXPath = doc.evaluate(soughtXPath, doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext().textContent;
+	newItem.title = evalXPath;
+	
+		// Authors
+	var authorCounter = 0;
+	var authorArray = new Array();
```

Comment:
why not grab author and date from the meta header without this ugly (and fragile) scraping? It looks like they provide that nicely:

```
 <meta property="article:published_time" content="2015-11-24T21:48:47Z" />
  <meta name="author" content="Philip Kollar" />
```
(from http://www.polygon.com/2015/11/24/9793694/league-of-legends-crafting-loot-system-toxic-players-hextech-2016-update)